cmd/acra-connector/acra-connector.go

```diff
@@ -404,7 +404,7 @@ func main() {
 				log.WithError(err).Fatalln("Cannot create OCSP config")
 			}
 
-			crlConfig, err := network.NewCRLConfig("", "use")
+			crlConfig, err := network.NewCRLConfig("", "use", 16)
```

Comment:
I'd leave some TODO here to make it configurable later.

Also, I believe AcraTranslator needs one too.